docs/Linting_Prettier.md

```diff
@@ -0,0 +1,107 @@
+
+The `.eslintrc.js` file is our [configuration file](https://eslint.org/docs/latest/use/configure/configuration-files).
+
+Since we are using the config file itself, the default `esLintConfig` section in the `package.json` file was removed. It originally used react-app and Jest:
+
+```
+"eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  }
+```
+
+**NOTE:** you can disable lint checks by following [this](https://eslint.org/docs/latest/use/configure/rules#disabling-rules).
+
+### Configuration Notes
+
+As per [PR #42](https://github.com/codefordallas/codefordallas.github.io/pull/42), this web app was built in conjunction with [create-react-app](https://create-react-app.dev/). [Here](https://create-react-app.dev/docs/setting-up-your-editor/#extending-or-replacing-the-default-eslint-config) is additional information on what is needed for the ESLint config file in relation to this app setup.
+
+**NOTE:** there are also setups for IDE/code editors, so be sure to check out these options as they relate to your environment setup.
+
+### Variable Explanations
+
+The `env` variable is generally used to [specify environments](https://eslint.org/docs/latest/use/configure/language-options#specifying-environments) like:
+- implements [es2021](https://eslint.org/docs/latest/rules/logical-assignment-operators), which includes operator assignments
+- tells linter that it will be run in the browser
+
+The `extends` variable allows us to [extend the config file](https://eslint.org/docs/latest/use/configure/configuration-files#extending-configuration-files). We have:
+- `react-app` (more on this [here](https://stackoverflow.com/a/59633134/10474024))
+- [prettier](https://github.com/prettier/eslint-plugin-prettier#installation)
+
+In the additional `plugins` section (more on plugin configuration can be found [here](https://eslint.org/docs/latest/use/configure/plugins)):
+- [jsx-ally](https://www.npmjs.com/package/eslint-plugin-jsx-a11y) (this could instead have been an extension as per [here](https://www.npmjs.com/package/eslint-plugin-jsx-a11y))
+
+<u>Prior</u> extensions used (should you want to review for your own projects):
+- [eslint:recommended](https://eslint.org/docs/latest/use/configure/configuration-files#using-eslintrecommended) for what the linter suggests
+- a [plugin](https://eslint.org/docs/latest/use/configure/configuration-files#using-a-configuration-from-a-plugin) for React recommendations
+
+The `parserOptions` allows us to configure a [custom parser](https://eslint.org/docs/latest/use/configure/parser). We currently have ours set to latest `ecmaVersion` and using [module syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) for `sourceType`.
+
+# Future Recommendations
+
+May want to consider integrating [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) or even [prettier-eslint](https://github.com/prettier/prettier-eslint).
```

Comment:
This should be done! I chose to go with just using the eslint extension here: https://github.com/codefordallas/codefordallas.github.io/blob/v1-mvp/.eslintrc.js#L2